lib/diagnostic.rb

```diff
@@ -45,7 +56,13 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Developer < Person
+  def initialize(name, age, location)
+  end
+  def solve_problems
+    puts "think think think"
+  end
+end
```

Comment:
Correct, however you don't need to initialize name, age, and location since it's already inheriting from Person & Person has those properties.